modules/kmm-applying-tolerations-to-kernel-module-pods.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="kmm-applying-tolerations-to-kernel-module-pods_{context}"]
+= Applying tolerations to kernel module pods 
+
+Taints and tolerations consist of `effect`, `key`, and `value` parameters. Tolerations include additional `operator` and `tolerationSeconds` parameters. 
+
+`effect`:: Indicates the taint effect to match. Empty means match all taint effects. When specified, allowed values are  `NoSchedule`, `PreferNoSchedule`, or `NoExecute`.
+
+`key`:: The taint key that the toleration applies to. Empty means match all taint keys. If the key is empty, `operator` must be `Exists`. This combination means to match all values and all keys.
+
+`value`:: The taint value the toleration matches to. If `operator` is `Exists`, the value should be empty, otherwise just a regular string.
```

Comment:
```suggestion
`value`:: The taint value the toleration matches to. If `operator` parameter is `Exists`, the value should be empty, otherwise just a regular string.
```

Reply:
I don't know what `otherwise just another string` means in this context.

Reply:
Changing this to `otherwise use a regular string`.